TypeScript/channelType.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * This plugin checks if a channel is the specified type
+ *
+ * @author @NeoYaBoi [<@762918086349029386>]
+ * @version 1.0.0
+ * @example
+ * ```ts
+ * import { channelType } from "../plugins/channelType";
+ * import { channelType } from "discord.js"

```

Comment:
dude fine, dont import it as type, just saw the conversation
but its still not `channelType` that was my main reason, djs doesnt export channelType, it exports ChannelType
```suggestion
 * import { ChannelType } from "discord.js"

```
![image](https://user-images.githubusercontent.com/85353424/185537755-c33be9ab-dda5-4687-86d4-154b3bf84847.png)